infra/docker/metagraph-ubuntu/Dockerfile

```diff
@@ -27,7 +28,7 @@ RUN git clone https://github.com/Constellation-Labs/tessellation.git && \
     cd tessellation && \
     git checkout $CHECKOUT_TESSELLATION_VERSION && \
```

Comment:
it is probably worth updating `git checkout` to `git switch` as this would've caught the error David was running into